sklearn/neighbors/base.py

```diff
@@ -421,6 +454,174 @@ class from an array representing our data set and ask who's
                 return dist, neigh_ind
             return neigh_ind
 
+    def masked_kneighbors(self, X=None, n_neighbors=None, return_distance=True,
```

Comment:
Do you mean the method masked_kneighbors itself? I thought you wanted me to create a separate method to handle missing datasets? Initially, I had the missing handling within kneighbors() but I created this later. Maybe I misunderstood you then?